l5kit/l5kit/dataset/select_agents.py

```diff
@@ -181,7 +180,7 @@ def select_agents(
         pass  # group is already there
 
     agents_mask = zarr.open_array(
-        str(Path(zarr_dataset.path) / "agents_mask" / output_group),
+        str(Path(zarr_dataset.path) / "agents_mask" / f"{th_agent_prob}"),
```

Comment:
`f"{zarr_dataset.path}/agents_mask/{th_agent_prob}"` looks simpler

Reply:
Since the filename has been used at two places, LINE161, LINE183, we can define a variable for it

Reply:
ok for the variable, but I will keep it in Path as ` f"{zarr_dataset.path}/agents_mask/{th_agent_prob}"` is platform specific.